docs/pages/1-generate-https.md

```diff
@@ -176,9 +176,9 @@ This will save all of the certificate files to /etc/letsencrypt/live/expfactory.
 Now we'll move these to where they're expected later.
 
 ```
-cp /etc/letsencrypt/live/myexpfactory.dynu.net/fullchain.pem /etc/ssl/certs/chained.pem
-cp /etc/letsencrypt/live/myexpfactory.dynu.net/privkey.pem /etc/ssl/private/domain.key
-cp /etc/letsencrypt/ssl-dhparams.pem /etc/ssl/certs/ssl-dhparams.pem
+cp /etc/letsencrypt/live/expfactory.dynu.net/fullchain.pem /etc/ssl/certs/chained.pem
+cp /etc/letsencrypt/live/expfactory.dynu.net/privkey.pem /etc/ssl/private/domain.key
+cp /etc/letsencrypt/ssl-dhparams.pem /etc/ssl/certs/dhparam.pem
 ```
```

Comment:
most will definitely need sudo here! e.g.,

```
sudo cp <certificate> /etc/ssl/certs/<location>
```
quick question - is there a way to write the docs so that the letsencrypt path is variable, so the user could just copy it?